crates/router/src/core/payments/helpers.rs

```diff
@@ -1157,7 +1157,7 @@ pub(crate) fn validate_payment_status(
         || {
             Err(errors::ApiErrorResponse::PreconditionFailed {
                 message: format!(
-                    "You cannot {action} this Payment because it has already {intent_status}",
+                    "You cannot {action} this payment because it has already {intent_status}",
```

Comment:
It might be better to use something like:

```"You cannot {action} this payment because it has status `{intent_status}`"```

I feel `[...] it has already confirmed` or `[...] it has already voided` (just taking a few examples) would make the sentence grammatically incorrect.

I'll leave this up to you, go with whatever you see fit.

Reply:
adding a been would make it gramatically correct, `[...] it has already been confirmed`